pandas/tests/groupby/test_quantile.py

```diff
@@ -248,6 +248,15 @@ def test_groupby_quantile_skips_invalid_dtype(q):
     tm.assert_frame_equal(result, expected)
 
 
+def test_groupby_quantile_NA(any_float_dtype):
```

Comment:
can you add cases with a) Int64 dtype b) all-NA column/group

Reply:
added